Detector/DetFCChhTrackerTkLayout/src/TkLayoutEndcap_Geo.cpp

```diff
@@ -126,6 +134,14 @@ static DD4hep::Geometry::Ref_t createTkLayoutTrackerEndcap(DD4hep::Geometry::LCD
   PlacedVolume placedDiscVolume = envelopeVolume.placeVolume(discVolume, DD4hep::Geometry::Position(0, 0, currentZ));
   placedDiscVolume.addPhysVolID("disc", discCounter);
   ++discCounter;
+  Acts::ActsExtension::Config layConfig;
+  // the local coordinate systems of modules in dd4hep and acts differ
+  // see http://acts.web.cern.ch/ACTS/latest/doc/group__DD4hepPlugins.html
+  layConfig.axes = "XZY"; // correct translation of local x axis in dd4hep to local x axis in acts
```

Comment:
To be honest, I'm not sure I understand. So I guess some illustrations might help indeed. For what I do understand, the axes conventions in ACTS are different for some shapes w.r.t. to TGeo. This raises a naive question: Wouldn't it be better / easier to simply use the type information in the  TGeo/DD4hep plugin instead of letting the user set this? (E.g. through templated translation functions.)

Anyway, this is something to be discussed within ACTS, and isn't in the way of mergin this PR, I think.